webviz_subsurface/_providers/ensemble_table_provider/ensemble_table_provider.py

```diff
@@ -1,9 +1,15 @@
 import abc
+from dataclasses import dataclass
 from typing import List, Optional, Sequence
 
 import pandas as pd
 
 
+@dataclass(frozen=True)
+class TableVectorMetadata:
```

Comment:
Related to comment associated with `EnsembleTableProvider.vector_metadata()`, maybe reconsider the name of this class?